### DISCUSSION TITLE
How to remove some of the detection layers?

### DISCUSSION THREAD

**Discussion by Yegor-R** [ultralytics/yolov5#4219] [Q&A]
Hi.

Is it possible to remove p5 layer to increase the speed of yolov5? 

In our data we only have tiny objects and I guess "large scale" detector will be useless.

**Comment by glenn-jocher** [comment]
@Yegor-R you're free to add or remove any inputs to the Detect() layer here:
https://github.com/ultralytics/yolov5/blob/c2c958c350407b630bbbf063cefbd64cea7d8c81/models/yolov5s.yaml#L45

Your understanding of speed impacts is not correct though. Smaller stride layers are by far the slowest (P3), not the largest stride layer (P5).
